web/src/components/app.tsx

```diff
@@ -280,7 +281,11 @@ let LocalizedPage: any = class extends React.Component<
                   }
                 />
               ))}
-              <Layout />
+              {location.pathname.includes('/demo') ? (
```

Comment:
I see that you've added the `/demo` path to the `URLS` constant, can you replace these strings with a reference to `URLS`?